docs/src/guide/use.md

```
@@ -1 +1,178 @@
 # TinyAgent 如何使用
+
+### 使用MCP server
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)